documentation/docs/steps/npmExecute.md

```diff
@@ -0,0 +1,23 @@
+# npmExecute
+
+## Description
+
+Content here is generated from corresponnding step, see `vars`.
+
+## Parameters
+
+Content here is generated from corresponnding step, see `vars`.
+
+## Step configuration
+
+Content here is generated from corresponnding step, see `vars`.
```

Comment:
What?

Reply:
This is a placeholder, which will be replaced with documentation in groovy files. Currently it is used in following files:
```
[INFO] Step 'uiVeri5ExecuteTests' has been rendered.
[INFO] Step 'checkChangeInDevelopment' has been rendered.
[INFO] Step 'dockerExecuteOnKubernetes' has been rendered.
[INFO] Step 'npmExecute' has been rendered.
[INFO] Step 'dockerExecute' has been rendered.
```
We discussed this and will solve this by improving the generator.